### PR TITLE
Add aria-controls to the relevant nav components of the viewer

### LINKED ISSUE #1670: Add aria-controls to the buttons for the media viewer
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-controls

### PR DESCRIPTION
**Pull request by aaron-collier**
Fixes #1670

There may be a few more buttons that need these values. I can put in follow up PRs if this is reviewed/merged before I identify any more controls.

### CODE REVIEW

File: app/components/embed/media_with_companion_windows_component.html.erb [hunk 1]
```diff
@@ -34,35 +34,35 @@
     </div>
   </div>
 
-  <div class="media-component-body">
+  <div class="media-component-body" id="left-drawer">
```
Comment (jcoyne) [inline app/components/embed/media_with_companion_windows_component.html.erb:37]: This is more than the left drawer.

**Review by jcoyne — CHANGES_REQUESTED** [review]

File: app/components/embed/media_with_companion_windows_component.html.erb [hunk 2]
```diff
@@ -1,14 +1,14 @@
 <div class="sul-embed-container media-component" id='sul-embed-object' style='display:none;' data-controller="media media-tag">
   <header>
     <nav aria-label="Window navigation">
-      <button data-action="click->media#toggleLeft">
+      <button data-action="click->media#toggleLeft" aria-label="Display left menu" aria-controls="left-drawer">
```
Comment (jcoyne) [inline app/components/embed/media_with_companion_windows_component.html.erb:4]: Is calling out "left" important here for someone who may be visually impaired? Is there a better description of this drawer we could use @alundgard ?

File: app/components/embed/media_with_companion_windows_component.html.erb (hunk 1 above)
Comment (aaron-collier) [reply app/components/embed/media_with_companion_windows_component.html.erb:37]: Sure, I don't think in this context it's being used informatively, it's just the link to the `aria-controls` value, but perhaps I'm misunderstanding.

Comment (jcoyne) [reply app/components/embed/media_with_companion_windows_component.html.erb:37]: I'm saying that this is the parent container of the left drawer and contains the media player and the right drawer too.

Comment (aaron-collier) [reply app/components/embed/media_with_companion_windows_component.html.erb:37]: Ah, gotcha.. Yeah, that's not right, I probably just put this in the wrong place, apologies.

**Review by jcoyne — APPROVED** [review]

**Pull request merged**